Docker/Dockerfile

```diff
@@ -1,12 +1,14 @@
-ARG BUILD_MODE="release"
-FROM lukemathwalker/cargo-chef:latest-rust-1.60.0-slim as builder
-
 # Dictates whether the underlying node should be built as a dev or
 # production binary. If empty, then the development mode will be built.
 # If passed as BUILD_MODE=--release, then the production will be built.
-ARG BUILD_MODE
+ARG BUILD_MODE="release"
 
-RUN echo "Will build image with mode: ${BUILD_MODE}"
+#################################################################
+# Stage 0
+#################################################################
+FROM lukemathwalker/cargo-chef:latest-rust-1.60.0-slim as chef
```

Comment:
Probably worth using the rust container directly instead of the cargo-chef one since we don't use cargo-chef